test/it/ArrivalSpec.scala

```diff
@@ -32,21 +33,23 @@ class ArrivalSpec extends IntegrationSpec {
 
   "Consignment References Page" when {
     "GET" should {
-      "return 200" in {
+      "throw feature disabled" in {
         // Given
         givenAuthSuccess("eori")
         givenCacheFor("eori", ArrivalAnswers())
 
         // When
         val response = get(controllers.routes.ConsignmentReferencesController.displayPage())
 
-        // TThen
-        status(response) mustBe OK
+        // Then
+        intercept[RuntimeException] {
+          status(response)
+        } mustBe FeatureDisabledException
```

Comment:
Here I think better is use:
```
intercept[FeatureDisabledException] {
    await(response)
}
```

`await` is from play framework so you don't need to extends any additional traits etc. Additionally the reason of using the `status` is to check the status, here we need to just wait for the result. Of course actual implementation works as expected.

Reply:
And additionally it's good to know that the result of the intercept is an exception, so if you need to validate a message from exception you can put the result to the val and check the message.

This is the reason that using `mustBe` here is working correctly